extensions/scarb-doc/src/main.rs

```diff
@@ -100,6 +108,22 @@ fn main_inner(args: Args, ui: Ui) -> Result<()> {
                     .unwrap_or(&pkg_output_dir)
                     .to_string();
                 ui.print(Status::new("Saving output to:", &output_path));
+                if args.build {
+                    let build_output_dir = pkg_output_dir.join("book");
+                    ScarbCommand::new()
+                        .arg("mdbook")
+                        .arg("--input")
+                        .arg(pkg_output_dir)
+                        .arg("--output")
+                        .arg(build_output_dir.clone())
+                        .env("SCARB_UI_VERBOSITY", ui.verbosity().to_string())
+                        .run()?;
```

Comment:
did you run it without issues in your local env? I am trying to do that. Yet, command exits with `error: no such command: mdbook`. However, scarb-`metadata/src/command/scarb_command.rs` `cmd.status()` is  `Ok(ExitStatus(unix_wait_status(256))) ` so if i change the line` if cmd.status()?.success() {` to `if cmd.status().is_ok() {` it builds the book as expected. Any idea what could be wrong with my local env or if is it not env specific?

Reply:
Hmm, how do you compile it? Sounds like scarb-mdbook has not been compiled. Can you try `cargo build --workspace --exclude scarb-prove --exclude scarb-verify`? Note there is no way in Cargo to declare that scarb-doc depends on scarb-mdbook binary (neither in tests nor in binary build https://github.com/rust-lang/cargo/issues/9096).

Reply:
Ok, it worked 👌  Setting `$SCARB` env did the job :)